.travis.yml

```diff
@@ -5,6 +5,18 @@ jdk:
   - oraclejdk8
 sudo: required
 dist: trusty
+cache:
+  directories:
+    - $HOME/mvn-home
+
+install:
+  # install maven 3.3.9 if it is not in cache already
```

Comment:
right, consistency and built-in testing for the stated minimum Maven level. probably no big deal either way.